c2cgeoportal/scripts/themev1tov2.py

```diff
@@ -63,27 +63,84 @@ def main():  # pragma: no cover
 
     # must be done only once we have loaded the project config
     from c2cgeoportal.models import DBSession, \
-        LayerInternalWMS, LayerExternalWMS, LayerWMTS, LayerV1
+        ServerOGC, LayerWMS, LayerWMTS, LayerV1
 
     session = DBSession()
-    session.execute(LayerInternalWMS.__table__.delete())
-    session.execute(LayerExternalWMS.__table__.delete())
-    session.execute(LayerWMTS.__table__.delete())
 
-    for layer in DBSession.query(LayerV1).all():
+    table_list = [LayerWMTS, LayerWMS, ServerOGC]
+    for table in table_list:
+        print "emptying table %s" % str(table.__table__)
+        # must be done exactly this way othewise the cascade config in the
+        # models are not used
+        for t in session.query(table).all():
+            session.delete(t)
+
+    # list and create all distinct server_ogc
+    server_ogc(session)
+
+    print "converting layerv1"
```

Comment:
mmm, without feedback, the user doesn't know whats being done, so I though a bit of text would be welcome, but it's not a necessity.